pkg/gitops/gitops.go

```diff
@@ -817,33 +809,8 @@ func CreateGitOpsCommit(gitOpsConfig *GitOpsConfig, appSlug string, appName stri
 	return gitOpsConfig.CommitURL(updatedHash.String()), nil
 }
 
-func generateKeyPair_RSA() (*KeyPair, error) {
-	privateKey, err := getPrivateKey()
-	if err != nil {
-		return nil, errors.Wrap(err, "get private key")
-	}
-
-	var publicKey *rsa.PublicKey
-	publicKey = &privateKey.PublicKey
-
-	privateKeyBytes := x509.MarshalPKCS1PrivateKey(privateKey)
-	PrivateKeyPEM := pem.EncodeToMemory(
-		&pem.Block{
-			Type:  "RSA PRIVATE KEY",
-			Bytes: privateKeyBytes,
-		},
-	)
-
-	publicKeySSH, err := ssh.NewPublicKey(publicKey)
-	if err != nil {
-		return nil, errors.Wrap(err, "convert public key to ssh")
-	}
-	pubKeySSHBytes := ssh.MarshalAuthorizedKey(publicKeySSH)
-
-	return &KeyPair{PrivateKeyPEM: string(PrivateKeyPEM), PublicKeySSH: string(pubKeySSHBytes)}, nil
-}
-
 func generatePrivateKey_ed25519() (*KeyPair, error) {
+	r := rand.New(rand.NewSource(time.Now().UTC().UnixNano()))
```

Comment:
this already gets initialized in the `init` function for the `gitops` package in the `key.go` file